typography/main.scss

```diff
@@ -1,6 +1,3 @@
-$o-typography-is-silent: false;
-$o-typography-load-fonts: false;
```

Comment:
might still need this second line. Not sure. Important to get it right though

Reply:
nope we’re safe—that variable is [only used in conjunction with the first](https://github.com/Financial-Times/o-typography/blob/master/main.scss#L26)

Reply:
:+1: